modules/es-extensions/store/extensions/app/greg_swagger/app.js

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2014, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+app.server = function(ctx){
```

Comment:
Thanks @splinter for pointing out this formatting issue, I have fixed it in [this commit](https://github.com/wso2/product-greg/pull/854/commits/bfa37074378e690eb2cb40ecf3d5de50e684f9e3)